instructionAPI/src/AMDGPU/gfx908/InstructionDecoder-amdgpu-gfx908.h

```diff
@@ -73,7 +74,11 @@ namespace Dyninst {
             static const char* bitfieldInsnAliasMap(entryID);
             static const char* condInsnAliasMap(entryID);
 
-
+            //Check if the index (2nd arg) is valid for the array (1st arg)
+            template <typename ArrayType, std::size_t n, typename IndexType>
```

Comment:
I don't see any usages of this. Should it be in this commit?

Reply:
There are many in `instructionAPI/src/AMDGPU/gfx908/amdgpu_gfx908_decoder_impl.C`.

Reply:
Oh, derp. I didn't notice that file had huge changes and wasn't expanded.